data_structures/min_ele_stack_const_time.cpp

```diff
@@ -0,0 +1,139 @@
+/**
+ * @author [S-M-J-I](https://github.com/S-M-J-I)
+ * @file
+ *
+ * Implementation of stack that retrieves the minimum element in constant time.
+ * (https://www.baeldung.com/cs/stack-constant-time)
+ *
+ * @details
+ * The MinStack data structure keeps track of the minimum element present in a
+ * stack and retrieves them at O(1) time.
+ */
+#include <cassert>
+#include <iostream>
+#include <stack>
```

Comment:
document headers please
 https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/CONTRIBUTING.md

Reply:
@realstealthninja documented the headers

Reply:
@realstealthninja merged the branches